nomos-core/chain-defs/src/da/blob/mod.rs

```diff
@@ -12,7 +12,7 @@ pub trait Blob {
 
 pub trait BlobSelect {
     type BlobId: info::DispersedBlobInfo;
-    type Settings: Clone;
+    type Settings: Clone + Send;
```

Comment:
Is this bound and the one in txSelect necessary? I dont think so. In this case you may want to add the bound where it is really used.